c7n/resources/secretsmanager.py

```diff
@@ -137,3 +142,98 @@ class MarkSecretForOp(TagDelayedAction):
                   op: tag
                   days: 1
     """
+
+
+@SecretsManager.action_registry.register('delete')
+class DeleteSecretsManager(BaseAction):
+    """Delete a secret and all of its versions.
+    The recovery window is the number of days from 7 to 30 that
+    Secrets Manager waits before permanently deleting the secret
+    with default as 30
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: delete-unencrypted-secrets
```

Comment:
i dont think this filter is correct, according to aws documentation all secrets are encrypted, if a specific key is not specified it uses the default `aws/secretsmanager` key: 

> If you don't specify this value, then Secrets Manager uses the key aws/secretsmanager . If that key doesn't yet exist, then Secrets Manager creates it for you automatically the first time it encrypts the secret value.

Reply:
@PratMis ^

Reply:
ah gotcha, good catch

Reply:
Thanks @thisisshi